databases/week1/queries.sql

```diff
@@ -0,0 +1,59 @@
+--0:
+DESCRIBE task;
+-- to see all columns and their name.
+
+SELECT *
+from task;
+
+--1: 
+SELECT COUNT(*) AS 'Total tasks' 
+FROM task;
+ 
+
+--2: 
+SELECT COUNT(*) AS 'Tasks Without Valid Due Date' 
+FROM task
+WHERE due_date IS NULL;
+
+
+--3: 
+SELECT *
+FROM task 
+WHERE status_id = 'done';
```

Comment:
The status_id is a number not a varchar, if you run this you get an empty result. For doing a conditional where you compare the status as a varchar, you need to join the table task with the table status.